i18n/SigDigger_zh_CN.ts

```diff
@@ -2843,4 +2843,645 @@
+        <source>Two line element set (TLE)</source>
+        <translation type="finished">两行轨道数(TLE)</translation>
+    </message>
+    <message>
+        <location filename="../ui/FrequencyCorrectionDialog.ui" line="76"/>
+        <source>Details</source>
+        <translation type="finished">详情</translation>
+    </message>
+    <message>
+        <location filename="../ui/FrequencyCorrectionDialog.ui" line="109"/>
+        <source>No frequency corrections enabled</source>
+        <translation type="finished">无频率修正</translation>
+    </message>
+    <message>
+        <location filename="../ui/FrequencyCorrectionDialog.ui" line="174"/>
+        <source>Please input a valid TLE set</source>
+        <translation type="finished">请输入一个有效的两行轨道数据集</translation>
+    </message>
+    <message>
+        <location filename="../ui/FrequencyCorrectionDialog.ui" line="190"/>
+        <source>&amp;Satellite</source>
+        <translation type="finished">卫星</translation>
+    </message>
+    <message>
+        <location filename="../ui/FrequencyCorrectionDialog.ui" line="209"/>
+        <source>Orbit details</source>
+        <translation type="finished">轨道详情</translation>
+    </message>
+    <message>
+        <location filename="../ui/FrequencyCorrectionDialog.ui" line="230"/>
+        <source>Azimuth</source>
+        <translation type="finished">水平</translation>
+    </message>
+    <message>
+        <location filename="../ui/FrequencyCorrectionDialog.ui" line="242"/>
+        <source>0 º</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../ui/FrequencyCorrectionDialog.ui" line="249"/>
+        <source>Inclination</source>
+        <translation type="finished">倾斜度</translation>
+    </message>
+    <message>
+        <location filename="../ui/FrequencyCorrectionDialog.ui" line="261"/>
+        <source>0 km/s</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../ui/FrequencyCorrectionDialog.ui" line="273"/>
+        <source>0 Hz</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../ui/FrequencyCorrectionDialog.ui" line="280"/>
+        <source>VLOS speed</source>
+        <translation type="finished">视距内速度</translation>
+    </message>
+    <message>
+        <location filename="../ui/FrequencyCorrectionDialog.ui" line="292"/>
+        <source>No</source>
+        <translation type="finished">否</translation>
+    </message>
+    <message>
+        <location filename="../ui/FrequencyCorrectionDialog.ui" line="299"/>
+        <source>Period</source>
+        <translation type="finished">周期</translation>
+    </message>
+    <message>
+        <location filename="../ui/FrequencyCorrectionDialog.ui" line="323"/>
+        <source>Rise in XX:XX:XX</source>
+        <translation type="finished">在XX:XX:XX上升</translation>
```

Comment:
Just wondering, does this translation mean "rising at <time>" or "<ETA> to rise"? Because with this text what I mean the ETA for the satellite to rise, like, how much you have to wait before observing the satellite rising.

Reply:
Sorry, I don't quit get the point. Does 'XX:XX:XX' refer to a **time point** in the future just like ETA, or a **time span**, like 2 hour:15min:10sec, you have to wait.

Reply:
It is more like a time span, yes. "You have to wait XX:XX:XX to see the satellite rising". Rise means that the satellite goes from below horizon to above horizon (i.e. when its altitude becomes a positive angle)

Reply:
Forgot to say, altitude and elevation in the sky coordinate jargon both refer to the same angle. You will see in the literature "Azimuth and elevation" or "Azimuth and altitude".

Reply:
I got it. And I remember that I've translated 'Altitude' to '海拔' and not a angle, (line 736), Is it correct?